docs/settings.rst

```diff
@@ -26,9 +26,17 @@ Deprecated. Use ``SCRAPY_POET_RULES`` instead.
 SCRAPY_POET_RULES
 -----------------
 
-Default: ``None``
+Default: :meth:`web_poet.default_registry.get_rules()
+<web_poet.rules.RulesRegistry.get_rules>`
+
+.. warning::
+
+    Although ``SCRAPY_POET_RULES`` already has values set from the return value of
+    :meth:`web_poet.default_registry.get_rules() <web_poet.rules.RulesRegistry.get_rules>`,
+    make sure you call :func:`web_poet.consume_modules <web_poet.rules.consume_modules>`
+    inside your ``settings.py`` if you're using other rules from other external packages.
```

Comment:
What does external packages mean in this context? Does it mean "not in the same Scrapy project"?

Reply:
Added some text to clarify that the external packages contain page objects annotated with the `@handle_urls`